units/player.gd

```diff
@@ -32,6 +34,7 @@ func _physics_process(delta):
 	if Input.is_action_just_pressed("fire"):
 		print('fired!')
 		var p = Projectile.instantiate()
-		owner.add_child(p)
+		p.spawn(owner, projectile_effects)
+		# TODO:: This transform should probably be modifiable by projectile effects too?
 		p.transform = projectile_spawner.global_transform
```

Comment:
All this is doing currently is making sure the projectile doesn't move with the player as the player moves.
I.e. it moves relative to the global space, not relative to the player.

Definitely can do other changes to this transform afterwards though with effects 👍